static/app/views/replays/detail/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -74,7 +75,9 @@ function BreadcrumbItem({
           ) : null}
         </TitleContainer>
 
-        <Description title={description}>{description}</Description>
+        <Tooltip title={description} showOnlyOnOverflow>
```

Comment:
Previously it was expecting a string, now it uses `<Tooltip>` so that we can also hide the native tooltip when it is not overflowing.